### PR TITLE
Users device

### PR DESCRIPTION
**Pull request by jkokh**

### CODE REVIEW

**Review by EGreg — CHANGES_REQUESTED** [review]
Please test it with a debugger, all lines look at the values of variables

**Comment by EGreg** [comment]
Did you check everything yet?

File: platform/plugins/Users/web/js/UsersDevice.js [hunk 1]
```diff
@@ -55,7 +71,15 @@
 		 * @param {Boolean} callback Whether the user already has a subscription
 		 */
 		subscribed: function (callback) {
-			this.adapter.subscribed(callback);
+			this.getAdapter(function(adapter, err) {
```
Comment (EGreg) [inline platform/plugins/Users/web/js/UsersDevice.js:74]: Found more problems ... err should be first parameter, not second! Node-style. Change it everywhere.

**Review by EGreg — CHANGES_REQUESTED** [review]

File: platform/plugins/Users/web/js/UsersDevice.js (hunk 1 above)
Comment (jkokh) [reply platform/plugins/Users/web/js/UsersDevice.js:74]: ok

**Review by EGreg — APPROVED** [review]

**Pull request merged**